Gemfile

```diff
@@ -6,7 +6,7 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 ruby ">= 2.7.4"
 
 # Bundle edge Rails instead: gem "rails", github: "rails/rails", branch: "main"
-gem "rails", "~> 7.0.0"
+gem "rails", "~> 7.0.5"
```

Comment:
I need to check how we can ask dependabot to update the bundle but do not touch the Gemfile.  In case of security fix, it *may* make sense to update the Gemfile, but it this case this seems to be gratuitous and the Gemfile.lock file *is* the thing that allows reproducible builds.

Reply:
I opened #67 which should fix this. When merged, we will try to `@dependabot recreate` and see if it helps!

Reply:
cc @neomilium  :grin:

Reply:
@dependabot recreate